streamlineTools/content.js

```
@@ -1,34 +1,35 @@
-  // if (!commentHeader) {
-    // commentHeader = "";
-  // }
-// } catch (e) {
-  // console.log(e);
-// }
-// let commentHeader = ''
-// let code = "";
-// let testCode = "";
+/* STUB | PRICING + UTIL CONTENT SCRIPT */
 
 if(typeof commentHeader === "undefined"){
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)